src/metadata.cpp

```diff
@@ -2407,7 +2407,7 @@ void Metadata::InternalData::update_indexes(int protocol_version, const VersionN
   std::string index_name;
 
   KeyspaceMetadata* keyspace = NULL;
-  TableMetadata::Ptr table;
+  TableMetadata::Ptr table(NULL);
```

Comment:
No need to init it by NULL due to default constructor of the object:
`explicit SharedRefPtr(T* ptr = NULL)`
So, please undo the change.